src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,126 @@
 package com.theironyard.novauc;
 
+import java.util.*;
+import java.text.NumberFormat;
+import java.math.*;
+import java.util.ArrayList;
+
 public class Main {
 
-    public static void main(String[] args) {
-	// write your code here
-    }
-}
+    public static void main(String[] args) throws Exception {
+
+        // write your code here
+
+//
+         ArrayList options = new ArrayList();
+
+        System.out.println("Good Morning!");
+
+        System.out.println("What is your name?");
+
+        Scanner scanner = new Scanner(System.in);
+
+        String name = scanner.nextLine();
+
+        System.out.println(" Welcome," + name);
+
+        HashMap<String, Double> account = new HashMap<>();
+        account.put("Mary", new Double(300.59) );
+        account.put("Johnny", new Double(1500.36));
+        account.put("Walter", new Double(2252.39));
+        account.put("Tony", new Double(398));
+
+        System.out.println("Check if Name is in Hashmap: " + account.containsKey(name));
+        if(account.containsKey(name)){
+
+        } else
+        System.out.println("Would like to create an account? ");
```

Comment:
The if else statement from line 34-38 should do something if they are in the hashmap.  Like say hello and print their balance.  And if they aren't in the hashmap it should add them to the hashmap.  Also watch your brackets.  line 38 runs every time the program runs.